ga-ccd-definition/CaseEvent/User/UserEventsGAspec-CUI-nonprod.json

```diff
@@ -0,0 +1,21 @@
+[
+  {
+    "CaseTypeID": "GENERALAPPLICATION",
+    "ID": "NO_REMISSION_HWF_GA",
+    "Name": "No remission HwF",
+    "Description": "No remission HWF",
+    "PreConditionState(s)": "AWAITING_APPLICATION_PAYMENT;APPLICATION_ADD_PAYMENT",
+    "PostConditionState": "*",
+    "SecurityClassification": "Public",
+    "CallBackURLAboutToStartEvent": "${CCD_DEF_GEN_APP_SERVICE_BASE_URL}/cases/callbacks/about-to-start",
+    "CallBackURLAboutToSubmitEvent": "${CCD_DEF_GEN_APP_SERVICE_BASE_URL}/cases/callbacks/about-to-submit",
+    "CallBackURLSubmittedEvent": "${CCD_DEF_GEN_APP_SERVICE_BASE_URL}/cases/callbacks/submitted",
+    "ShowSummary": "Y",
+    "ShowEventNotes": "N",
+    "EndButtonLabel": "Submit",
+    "RetriesTimeoutAboutToStartEvent": 0,
+    "RetriesTimeoutURLAboutToSubmitEvent": 0,
+    "RetriesTimeoutURLSubmittedEvent": 0,
+    "_EventEnablingCondition": "hwfFeeType = \"CLAIMISSUED\" OR hwfFeeType = \"HEARING\""
```

Comment:
I think the underscore was added for testing?

Reply:
> I think the underscore was added for testing?

removed. was copied from civil ccd but in ga this field is not filled by cui so deleted it